recipes/pygame/patches/0007-mark-docs-test-broken.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/test/docs_test.py b/test/docs_test.py
+index de021a8eb4..8b58e2e318 100644
+--- a/test/docs_test.py
++++ b/test/docs_test.py
```

Comment:
Generally I'd prefer proper patches produced by `git format-patch`, with author & commit-message.

(goes for the other patches below as well)

Reply:
Will do that if there are any more patches, but I would need to re-apply all changes, which would not be fun and use up time I don't have ;).

Reply:
This is why it's good to have a local source checkout with a branch of all the conda-patches. Makes it easy to go back and forth, and recreate the set of patches on demand 🙃

Reply:
So the patches were pretty mangled. I once a bit of [mini-guide](https://github.com/conda-forge/tensorflow-feedstock/pull/203#pullrequestreview-874021990) for how to work with many patches across versions (which I got shown resp. picked up from Ray Donnelly at the python-feedstock).

In particular, it should now be possible to re-apply them to a fresh pygame checkout just by a single command, e.g.
```
ls -d ../path/to/feedstock/recipe/patches/ | xargs git am
```

Let's please try to keep them git-applicable like that. :)